test/unit/services/subscription/EnrolmentStoreProxyServiceSpec.scala

```diff
@@ -113,6 +116,40 @@ class EnrolmentStoreProxyServiceSpec extends UnitSpec with MockitoSugar with Bef
 
       verify(mockEnrolmentStoreProxyConnector).getEnrolmentByGroupId(any[String])(meq(headerCarrier), any())
     }
+
+    "return Existing EORI" when {
+
+      "EORI is allocated to different groupId" in {
+
+        val es1Response = ES1Response(Some(Seq("groupId")), None)
+
+        when(mockEnrolmentStoreProxyConnector.queryGroupsWithAllocatedEnrolment(any())(any()))
+          .thenReturn(Future.successful(es1Response))
+
+        val existingEori = ExistingEori("Gb123456789123", "HMRC-GVMS-ORG")
+
+        val result = service.isEnrolmentInUse(Service.withName("atar").get, existingEori)
+
+        result.futureValue shouldBe Some(existingEori)
+      }
+    }
+
+    "not return existing EORI" when {
+
+      "EORI is not allocated to different groupId" in {
```

Comment:
eori is not used for the service the user is attempting to enrol

Reply:
Done